File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)

{
int n;
 do
   {
   n=get_int("number of blocks? ");
   }
 while (n<1);
for (int i=0; i<n; i++)
 {
    for (int y=0; y<=i; y++)
    {
     printf("#");
    }
   printf("\n");
 }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    try:
        height = int(input("Height: "))
        if (height >= 1) and (height <= 8):
            break
    except:
        print("", end="")

spaces = 1
# prints newline
for j in range(height):

    # prints spaces
    for spaces in range(height-j-1):
        print(" ", end="")

    # prints hashes
    for i in range(j+1):
        print("#", end="")
    print("  ", end="")
    for k in range(j+1):
        print("#", end="")

    print()
